### PR TITLE
integrations: add rekor-sidekick

### PR DESCRIPTION
**Pull request by srenatus**
This is a cool project. But for a proper entry, I'll need some assistance from @nsmith5: 

- [ ] Is there an organization behind this we should link to?
- [ ] Do you think the description and categorization are accurate or would you like to see it edited?

### CODE REVIEW

**Comment by nsmith5** [comment]
Yup this description sounds good to me @srenatus. Rekor sidekick is personal project, but my employer [chainguard](https://chainguard.dev/) helps me keep it maintained so you can add them as a supporting organization if you like.

File: build/policy/files.rego [hunk 1]
```diff
@@ -9,10 +9,11 @@ package files
 
 import future.keywords.in
 
-import data.helpers.endswith_any
-import data.helpers.last_indexof
+import data.helpers.extension
+import data.helpers.directory
 
 filenames := {f.filename | some f in input}
+logo_exts := {"png", "svg"}
```
Comment (anderseknert) [inline build/policy/files.rego:16]: We should update the docs to reflect this:
https://github.com/open-policy-agent/opa/blob/35184350b5380bfa63bd63688ee3a24d07330856/docs/README.md?plain=1#L334

**Review by anderseknert — COMMENTED** [review]
Nice! Some nits in the comments but looks good 👍

File: build/policy/files.rego [hunk 2]
```diff
@@ -32,20 +33,19 @@ deny["Logo must be placed in docs/website/static/img/logos/integrations"] {
 	"docs/website/data/integrations.yaml" in filenames
 
 	some filename in filenames
-	endswith(filename, ".png")
+	print(extension(filename))
```
[the remark this answers is not in the document]
Comment (srenatus) [reply build/policy/files.rego:36]: Haha
Will fix.

File: build/policy/helpers.rego [hunk 3]
```diff
@@ -10,7 +10,6 @@ last_indexof(string, search) = i {
 	true
 }
 
-endswith_any(string, suffixes) {
-	some suffix in suffixes
-	endswith(string, suffix)
-}
+extension(filename) = substring(filename, last_indexof(filename, ".")+1, count(filename)-1)
```
[the remark this answers is not in the document]
Comment (anderseknert) [reply build/policy/helpers.rego]: 😆 but we already have CI checks (i.e tests) running for these policies, so running a formatter check would be a good addition IMO... I can add it later though.

**Comment by srenatus** [comment]
let's *rebase* and merge this one -- those two commits ended up being unrelated to each other.

**Review by anderseknert — APPROVED** [review]
LGTM!

**Comment by anderseknert** [comment]
Can we ignore the netlify failures?

**Pull request merged**